### PR TITLE
nixos/dysnomia: add types

### PR DESCRIPTION
**Pull request by Scriptkiddi**
###### Motivation for this change
fixes in the list of #76184

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: nixos/modules/services/misc/dysnomia.nix [hunk 1]
```diff
@@ -103,30 +103,36 @@ in
 
       properties = mkOption {
         description = "An attribute set in which each attribute represents a machine property. Optionally, these values can be shell substitutions.";
+        type = types.attrsOf types.str;
         default = {};
       };
 
       containers = mkOption {
         description = "An attribute set in which each key represents a container and each value an attribute set providing its configuration properties";
+        type = types.attrsOf types.attrs;
```
Comment (SuperSandro2000) [inline nixos/modules/services/misc/dysnomia.nix:112]: Should this be more specific?

Comment (Scriptkiddi) [reply nixos/modules/services/misc/dysnomia.nix:112]: I could go with:
types.attrsOf (types.attrsOf (types.either types.str types.int types.bool) 
what do you think?

Comment (SuperSandro2000) [reply nixos/modules/services/misc/dysnomia.nix:112]: I think that would be more useful despite being complex.

**Comment by aanderse** [comment]
Actually... we should just remove this module.

https://github.com/NixOS/nixpkgs/issues/72962

**Comment by Scriptkiddi** [comment]
I will open a pr

**Comment by Scriptkiddi** [comment]
#110799

**Pull request closed without merging**